.github/workflows/doc-build.yml

```diff
@@ -23,6 +23,9 @@ on:
     - '.github/workflows/doc-build.yml'
     - 'scripts/dts/**'
     - 'scripts/requirements-doc.txt'
+  pull_request_target:
+    paths:
+    - 'doc/**'
```

Comment:
`pull_request_target` is a source of security problems. We should not be using it unless it is strictly needed.